core/domain/feedback_jobs_one_off.py

```diff
@@ -66,3 +66,34 @@ def reduce(key, stringified_message_ids):
                 'next_message_id': next_message_id,
                 'message_count': len(message_ids)
             })
+
+
+class FeedbackSubjectOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for updating the feedback subject."""
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [feedback_models.FeedbackThreadModel]
+
+    @staticmethod
+    def map(item):
+        if item.subject != '(Feedback from a learner)':
```

Comment:
Consider pulling out the hardcoded string as a class-level constant, too.

Reply:
Done